_posts/2021-04-19-hello-world-blog.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Hello (Underwater-) World"
+date:   2021-04-19 14:35:00 +0100
+categories: introduction
+author:
+- Alexandra Settle
+---
+
+Welcome to Aquarium!
+
+If you want to get involved, look at the [issue](https://github.com/aquarist-labs/aquarium/issues) list or
+check out our Slack channel (see below) and ask one of the friendly contributors.
+You can also view our [project board](https://github.com/orgs/aquarist-labs/projects/3)
+and check our [Review Guidelines](CONTRIBUTING.md).
+
+If you want to get your hands dirty as soon as possible, you can also run the
+script at `tools/setup-dev.sh`. This will ensure you have a basic development
+environment as soon as possible so you can start hacking.
+
+You will be able to find the backend bits in `src/gravel`, while the frontend
+bits are located in `src/glass`. We also have `wasser` (water) powering our CI.
+
+Check out our [From Zero to Hacking](https://github.com/aquarist-labs/aquarium/blob/main/doc/from-zero-to-hacking.md)
+quickstart to help you get off your feet.
```

Comment:
off your feet?

Reply:
LOL we need to change that in about 4 documents. GO ME.